src/v2/Components/Footer.tsx

```diff
@@ -265,6 +266,12 @@ const Link = styled(RouterLink)`
   align-items: center;
   padding: ${space(1)}px 0;
 `
+const CCPAWrapper = styled.p`
+  display: flex;
+  text-decoration: none;
+  align-items: center;
+  padding: ${space(1)}px 0;
+`
```

Comment:
This is a good example of how we can lean on Palette components, versus writing manual css and html:

```tsx
<Flex pt={1} justifyContent='center'>
  <CCPARequest />
</Flex>
```